examples/1_simple_runbooks/backup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+from brigade.easy import easy_brigade
+from brigade.plugins.tasks import files, networking, text
+
+
+def backup(task):
+    result = task.run(networking.napalm_get,
+                      getters="config")
+
+    return task.run(files.write,
+                    content=result.result["config"]["running"],
+                    filename="./backups/{}".format(task.host))
+
+
+brigade = easy_brigade(
+        hosts="../hosts.yaml", groups="../groups.yaml",
+        dry_run=False,
+        raise_on_error=False,
+)
+
+# select which devices we want to work with
+filtered = brigade.filter(type="network_device", site="cmh")
+results = filtered.run(backup)
+
+# Let's print the result on screen
+filtered.run(text.print_result,
+             num_workers=1,  # we are printing on screen so we want to do this synchronously
+             data=results)
```

Comment:
Why would you need to do this `text.print_result` and not just something like `print(results.result)`?

Reply:
because the `text.print_result` adds format and color, it's kind of like the callback plugins in ansible. Will post a few screencasts when the examples are finished.

Reply:
You should probably do some examples without this as it is (IMO) overly complicated just to get colored output (i.e. don't force things through brigade when easy options already exist in Python).

Reply:
The tutorial covers it: http://brigade.readthedocs.io/en/rework_examples/tutorials/intro/running_tasks.html